### PR TITLE
py(setup.py): add bidi extra requirement for trio

### PR DESCRIPTION
**Pull request by blueyed**
ref: https://github.com/SeleniumHQ/selenium/issues/11915

TODO:
- [ ] doc

### CODE REVIEW

File: py/setup.py [hunk 1]
```diff
@@ -72,10 +72,14 @@
...
-        "trio~=0.17",
-        "trio-websocket~=0.9",
         "certifi>=2021.10.8",
     ],
+    'extras_require': {
+        "bidi": [
+            "trio~=0.17",
+            "trio-websocket~=0.9",
+        ],
+    },
```
Comment (blueyed) [inline py/setup.py:82]: Quoting style is mixed, but I've tried to follow the existing pattern.  Should I just use double quotes everywhere?

Comment (titusfortner) [reply py/setup.py:82]: @symonk what is our preference here?

Comment (blueyed) [reply py/setup.py:82]: black uses double quotes, which I also prefer.

Comment (AutomatedTester) [reply py/setup.py:82]: It would be double quotes but let's do that in a separate PR

**Comment by AutomatedTester** [comment]
How much of an issue is this in reality? We're going to be making BiDi the main way to do things in the future.

File: py/setup.py (hunk 1 above)
Comment (blueyed) [reply py/setup.py:82]: I can use just double quotes here already.

**Comment by diemol** [comment]
Closing as we never got feedback.

**Pull request closed without merging**